Riku/src/GameLogic/Assets/Asset.h

```diff
@@ -45,7 +45,7 @@ namespace logic {
 		}
 
 		bool hasFunction(const std::string& key) {return functions.find(key)!=functions.end();}
-		[[nodiscard]] const std::map<std::string, sol::function>& getFunctions() const {return functions;}
+		[[nodiscard]] const std::map<std::string, sol::function>& getFunctions() {return functions;}
```

Comment:
Skoro funkcja nie jest const, to czemu zwraca const reference, a nie zwykłą referencję?

Reply:
to czy funkcja jest const czy nie chyba mówi tylko, że nie może zmieniać obiektu, a nie ma związku ze zwracanym typem.

Reply:
Ta funkcja powinna być const. Przywróce go